paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerApiRepository.kt

```diff
@@ -90,4 +90,23 @@ internal class CustomerApiRepository @Inject constructor(
                 logger.error("Failed to detach payment method $paymentMethodId.", it)
             }.getOrNull()
         }
+
+    override suspend fun attachPaymentMethod(
+        customerConfig: PaymentSheet.CustomerConfiguration,
+        paymentMethodId: String
+    ): PaymentMethod? =
+        withContext(workContext) {
```

Comment:
`DefaultStripeNetworkClient` already uses an IO context here, so this isn’t needed.